src/myst.ts

```diff
@@ -36,6 +36,28 @@ import { imageUrlSourceTransform } from './images';
 import { internalLinksPlugin } from './links';
 import { addCiteChildrenPlugin } from './citations';
 
+export interface MySTOptions {
+
+  parserOptions: Partial<AllOptions>;
+}
+
+export interface MySTOptionsProvider<Widget> {
+
+  get(widget: Widget): MySTOptions;
+}
+
+export class MySTNotebookDefaults implements MySTOptionsProvider<StaticNotebook> {
+
+  get(notebook: StaticNotebook): MySTOptions {
+    return {
+      parserOptions: {
+        directives: [cardDirective, gridDirective, ...tabDirectives],
+        roles: [evalRole],
+      },
+    };
+  }
+}
```

Comment:
I would love to see you are overriding things with other extensions @tavin!

Reply:
Here: https://github.com/tavin/myst-options-poc/tree/main/src